<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controller</class>
 <widget class="QWidget" name="Controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pbConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pbClrLog">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QTextBrowser" name="tbLog"/>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pbInitCmds">
       <property name="text">
        <string>Cmd</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="cbCommands"/>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="leSshConn">
       <property name="text">
        <string>root@192.168.130.7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pbTest">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="leConfigFile">
       <property name="text">
        <string>config.cfg</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbLogLevel"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
